src/layouts/DashboardLayout.js

```diff
@@ -1,62 +1,81 @@
 import React from 'react';
-import { Layout, Menu, Icon } from 'antd';
+import { Layout, Menu, Icon, Dropdown } from 'antd';
 import { Link } from 'react-router-dom';
 import { injectIntl } from 'react-intl'; 
 import AppFooter from './AppFooter';
 import logo from '../assets/logo.png'
 import './DashboardLayout.less'
+import { UserContext } from '../UserContext';
 
 const { Content, Sider } = Layout;
 
+const onClick = ({ key }) => {
+  localStorage.removeItem('ws-token');
```

Comment:
Consider moving this part to Api.js. 'ws-token' should be a private thing in Api.js and shouldn't be used everywhere.